mobify/platform-scaffold
# Catch up with develop again

Update `amp` with latest from `develop`

~~**JIRA**: (link to JIRA ticket)~~

## Changes
- Update with latest changes from `develop`
- Unfortunately temporarily reinstate workaround for `progressive-web-sdk` web dependency (https://github.com/mobify/platform-scaffold/pull/854/commits/65d405d0d051c319e52bfccb0ffa54f9b6b84636)

## How to test-drive this PR
- Run `npm run dev`
- Check out http://localhost:3000/potions.html http://localhost:3000/eye-of-newt.html
